cvmfs/cloud_testing/run.sh

```diff
@@ -242,6 +247,12 @@ devel_package=$(read_package_map    ${ctu}/pkgmap "$platform" 'devel'    )
 unittest_package=$(read_package_map ${otu}/pkgmap "$platform" 'unittests')
 config_packages="$(read_package_map ${otu}/pkgmap "$platform" 'config'   )"
 
+# if we are on mac then we have to run an special script
```

Comment:
Getting the package name from the package map is one line of code that also just calls a wrapper function - that's fine to copy over. The parameter reading from `platforms.json` can be shared in `run_cloudtests.sh`. Only the final call to `run.sh` needs to be wrapped into a condition, no?